.github/workflows/reusable_multi_numa.yml

```diff
@@ -53,13 +53,20 @@ jobs:
         run: cmake --build ${{github.workspace}}/build -j $(nproc)
 
       - name: Run tests
-        if: matrix.os != 'rhel-9.1'
+        if: (matrix.os != 'rhel-9.1') && (matrix.os != 'sles15')
```

Comment:
I would run all tests on SLES - did it not work? can you send some logs with failing SLES on all tests?